lib/summary.js

```diff
@@ -60,12 +60,17 @@ class Summary {
 				help,
 				labelNames: labels,
 				percentiles: configurePercentiles(obj.percentiles),
-				registers: [globalRegistry]
+				registers: [globalRegistry],
+				maxAgeSeconds: obj.maxAgeSeconds,
+				ageBuckets: obj.ageBuckets
 			};
 		}
 
 		validateInput(config.name, config.help, config.labelNames);
 
+		this.maxAgeSeconds = config.maxAgeSeconds || 600;
```

Comment:
This is a breaking change, altering the current behaviour. Could the defaults be changed so it doesn't change the current behaviour?

Reply:
That's great but it's still a breaking change and I want to avoid them as much as possible if it's doable (which I think it is in this case)

Reply:
I get your point but I don't think it's worth a breaking change for this, however I think for the next major bump we probably should change the default to the sliding window instead.

Reply:
Alright I'll fix it :)